app/controllers/admin_controller.rb

```diff
@@ -26,6 +26,15 @@ def users
     end
   end
 
+  def delete_user
+    begin
+      render json: KuluService::API.new.delete_user(api_params(delete_user_params)).to_json
```

Comment:
` KuluService::API.new.delete_user` returns a boolean. Doing a `to_json` on a boolean will simply return a boolean back. Is that intended?

Can we simply not just `to_json` the response body and return the `response.body` from the `delete_user` method?

Reply:
@kitallis `delete_user` returns a 200 with `{:id id-of-the-record-deleted}` body. It's inline with the other deletes e.g. on invoices and categories.